Sources/AudioKit/Nodes/NodeStatus.swift

```diff
@@ -0,0 +1,18 @@
+// Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKit/
+
+/// Keep track of a node status
+public enum NodeStatus {
+    /// Node status for a playback node
+    public enum Playback {
+        /// The player node is stopped.
+        case stopped
+        /// The player node is playing.
+        case playing
+        /// The player node is paused.
+        case paused
+        /// The player node is scheduling.
+        case scheduling
```

Comment:
This replaces the old property `isScheduled` and there is no need for `scheduleTime` anymore since the only thing dependent on it was the `isScheduled` property.